server/src/test/java/io/spine/server/command/model/CommandAssigneeMethodTest.java

```diff
@@ -105,24 +105,23 @@ private static CommandEnvelope newCommand(CommandMessage msg) {
 
     @Nested
     @MuteLogging /* Signature mismatch warnings are expected. */
-    @DisplayName("invoke handler method which returns")
-    class InvokeHandlerMethod {
+    @DisplayName("invoke assignee method which returns")
+    class InvokeAssigneeMethod {
 
         @Test
         @DisplayName("one `Message`")
         void returningMessage() {
-            var handlerObject = new ValidHandlerTwoParams();
-
-            var createdMethod = new CommandHandlerSignature().classify(handlerObject.method());
-            assertTrue(createdMethod.isPresent());
-            var handler = createdMethod.get();
+            var assignee = new ValidAssigneeTwoParams();
+            var method = new CommandAssigneeSignature().classify(assignee.method());
+            assertTrue(method.isPresent());
```

Comment:
Could you please migrate this class to `Truth8.assertThat(someOptional).isPresent()`? Of course, `Truth8.assertThat` should be imported statically.